app/assets/javascripts/tests/_draw_session.test.js

```diff
@@ -16,8 +16,47 @@ test('clearOtherSessions removes all markers expect the selected one', t => {
   t.end();
 });
 
+test('drawMobileSession draws a session when session is loaded and sensor is selected', t => {
+  const map = mock('drawMarker');
+  const heat = { getLevel: () => 1 };
+  const sensors = selectedSensor;
```

Comment:
Have you considered inlining this in the obj?

Reply:
Have you considered not passing `sensors` in `_drawSession` and have a default inside `_drawSession`?